userspace/libsinsp/event.h

```diff
@@ -44,7 +44,7 @@ typedef enum filtercheck_field_flags
 typedef struct filtercheck_field_info
 {
 	ppm_param_type m_type; ///< Field type.
-	filtercheck_field_flags m_flags;  ///< Field flags.
+	uint32_t m_flags;  ///< Field flags.
```

Comment:
Gianluca changed those flags to be a bitmask a while ago, but since we never actually used more than one flag until `lattr`/`rattr` the compiler wasn't complaining. But since it's a bitmask it cannot be the enum type.

Reply:
Ok, can we get rid of the enum entirely then and have defines/consts for the enum values?

Reply:
Sure, will do